src/components/NcSelect/NcSelect.vue

```diff
@@ -983,15 +983,12 @@ export default {
 		},
 
 		propsToForward() {
-			const {
-				// Props handled by this component
-				inputClass,
-				noWrap,
-				placement,
-				userSelect,
-				// Props to forward
-				...initialPropsToForward
-			} = this.$props
+			const initialPropsToForward = { ...this.$props }
+			// Props handled by this component
+			delete initialPropsToForward.inputClass
+			delete initialPropsToForward.noWrap
+			delete initialPropsToForward.placement
+			delete initialPropsToForward.userSelect
```

Comment:
This seems unrelated, no?

Reply:
No this fixes ESLint which will complain about unused variables, because you create const variables `inputClass`, `noWrap` etc without using them.
So instead of creating unused variables, just delete the props we use.

Reply:
(BTW I personally even think the explicit delete is more readable meaning it makes it clear what is expected to happen)